pytest_rerunfailures.py

```diff
@@ -282,6 +282,17 @@ def _try_match_reprcrash(rerun_errors, report):
     return False
 
 
+def _matches_any_rerun_except_error(rerun_except_errors, report):
+    for rerun_regex in rerun_except_errors:
+        try:
+            if re.search(rerun_regex, report.longrepr.reprcrash.message):
+                return True
+        except AttributeError:
+            if re.search(rerun_regex, report.longreprtext):
+                return True
+    return False
+
+
```

Comment:
```suggestion
```

This issue was already fixed in #240. But I'd like to keep the test case.

Reply:
Hah, bad timing it seems :D